umbra-js/src/classes/Umbra.ts

```diff
@@ -745,8 +754,10 @@ async function recursiveGraphFetch(
     body: JSON.stringify({
       query: query(`
         first: 1000,
+        orderBy: timestamp,
+        orderDirection: desc,
         where: {
-          ${fromId ? `id_gt: "${fromId}",` : ''}
+          ${fromTimestamp ? `timestamp_lt: "${fromTimestamp}",` : ''}
```

Comment:
Why did we change from using the log ID to using timestamp? Not sure if it matters, just wondering. For reference, we have the IDs in the subgraph set to `event.block.number.toString() + '-' + event.logIndex.toString()`

Reply:
> subgraph set to event.block.number.toString() + '-' + event.logIndex.toString()

Does this mean the `formId`s are already ordered by their block number? if that's the case we can revert this change. The idea was just to start fetching from the most recent announcements to the oldest last.

Reply:
Correct, the ordering by ID should be equivalent to ordering by timestamp. Note that ordering by timestamp is _not_ unique because with batch sends you can have multiple logs in the same timestamp. Whereas ordering by this ID is deterministic because it uses block number and tx log index

Reply:
Oh true, that's good to know. Thanks!

Reply:
Yea so we should probably revert this back to using IDs, since I don't think timestamp will guarantee unique ordering (it might, but no reason to rely on the ambiguity there)

Reply:
For reference, you can find our subgraph / ID definition here: https://github.com/ScopeLift/umbra-subgraph/blob/a390b27fa6a6431875402e07b3ae302e87c59ec3/src/mapping.ts#L7

Reply:
So an additional change that was added was using `orderBy: timestamp` and `orderDirection: desc` whitch helped fetch the latest 1000 announcements. 
Seems like using `formId` as a filter to query makes it fetch the oldest 1k announcements. And me trying to use `orderBy : formId` is giving the following error ![Screenshot 2023-07-26 at 4 37 57 PM](https://github.com/ScopeLift/umbra-protocol/assets/61768337/11a5c9f8-3c99-4c76-ac88-140a341107f4)
Maybe we should break this into an issue because currently ordering by timestamps is working batchSend transactions.

Reply:
Issue https://github.com/ScopeLift/umbra-protocol/issues/578